pkgs/tools/admin/trivy/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "trivy";
+  version = "0.11.0";
+
+  src = fetchFromGitHub {
+    owner = "aquasecurity";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0mqzs0yjlqpsn2540kgl4gn77qjg3kfls1zwgjfxy66lazic3f1h";
+  };
+
+  vendorSha256 = "1jmmx83c0qbqyppx0w8d9djz2j2i7dgibq9lb3vhdz78wc40zywy";
+
+  buildFlagsArray = [
+    "-ldflags="
+    "-s"
+    "-w"
+    "-X main.version=v${version}"
+  ];
+
+  meta = with stdenv.lib; {
+    description = "A Simple and Comprehensive Vulnerability Scanner for Containers, Suitable for CI.";
+    longDescription = ''
+      Trivy is a simple and comprehensive vulnerability scanner for containers and other artifacts.
+      A software vulnerability is a glitch, flaw, or weakness present in the software or in an Operating System.
+      Trivy detects vulnerabilities of OS packages (Alpine, RHEL, CentOS, etc.) and application dependencies (Bundler, Composer, npm, yarn, etc.).
+    '';
+    homepage = src.meta.homepage;
```

Comment:
What??

Reply:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/video/kodi/plugins.nix#L255

https://search.nixos.org/packages?show=kodiPlugins.simpleplugin&query=simpleplugin&from=0&size=30&sort=relevance&channel=unstable#disabled

https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/fetchgithub/default.nix#L33

I could hardcode it if you want, but it works 🤷

Reply:
Ive also seen someone doing this but with `inherit`

Reply:
No problem. I am just wondering about this strange property of `fetchFromGitHub`.